doc/API.md

```diff
@@ -65,6 +66,15 @@ For a full listing of API offered by aXe, clone the repository and run `npm run
 - The `"helpUrl"` in the results object is a link to a broader description of the accessibility issue and suggested remediation. These links point to Deque University help pages, which do not require a login.
 - aXe does not test hidden regions, such as inactive menus or modal windows. To test those for accessibility, write tests that activate or render the regions visible and run the analysis again.
 
+#### Supported CSS Selectors
```

Comment:
I think we should move this to the developer guide. It's not relevant unless you're substantially tweaking axe-core.